lib/editor/util/map.js

```diff
@@ -44,11 +44,14 @@ type R5Response = {
   }>
 }
 
+export const coordIsOutOfBounds = (coords: LatLng, bounds: any) => {
+  return coords.lat > bounds.getNorth() ||
+    coords.lat < bounds.getSouth() ||
+    bounds.lng > bounds.getEast() ||
+    bounds.lng < bounds.getWest()
+}
 export const stopIsOutOfBounds = (stop: GtfsStop, bounds: any) => {
-  return stop.stop_lat > bounds.getNorth() ||
-    stop.stop_lat < bounds.getSouth() ||
-    stop.stop_lon > bounds.getEast() ||
-    stop.stop_lon < bounds.getWest()
+  return coordIsOutOfBounds({lat: stop.stop_lat, lng: stop.stop_lon})
```

Comment:
Do we need to pass `bounds` to `coordIsOutOfBounds`? I'm getting a typeError when running this.

Reply:
Yep slept right through that one thanks for noticing! I guess the thing I did to test that method was working wasn't working.